docs/user-guide/how-to.md

```diff
@@ -148,3 +195,5 @@ Please refer to it for complete examples and guides.
 [terraform-docs GitHub Action]: https://github.com/terraform-docs/gh-actions
 [git hooks]: https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks
 [pre-commit-terraform]: https://github.com/antonbabenko/pre-commit-terraform
+[pre-commit]: https://pre-commit.com/
```

Comment:
```suggestion
[`pre-commit`]: https://pre-commit.com/
```

Otherwise this doesn't appear as hyperlink.

Reply:
I'm not sure I understand. It matches all of the other links in the block, and those seem to work. Is there something different about mine?

Reply:
Yes, whatever goes inside `[ ]` has to match with footer's link. You can check this in action [here](https://deploy-preview-436--terraform-docs.netlify.app/user-guide/how-to/#pre-commit-hook) where instead of hyperlink it gets generated as literal "[`pre-commit`]".

Other option would be to name the link to prevent using backticks, something like:

```markdown
With [`pre-commit`][any-random-id], you can ensure your ...
...

[any-random-id]: https://pre-commit.com/
```

Reply:
Thanks, will fix!